experiment/pretest.json

```diff
@@ -0,0 +1,21 @@
+[{
+        "question": "The most effect method of IR spectroscopy is the:",
```

Comment:
Please fix grammatical errors.

Reply:
Still not fixed

Reply:
Sir, It is fixed. I've corrected options according to the question. Please check the options once.

Reply:
Should it be 'the most effect method' or 'the most effective method'?